media-sound/ladish/ladish-1.1.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+PYTHON_REQ_USE='threads(+)'
+
+inherit python-single-r1 waf-utils
+
+MY_P="${P}-g36c489e4"
+DESCRIPTION="LADI Session Handler - a session management system for JACK applications"
+HOMEPAGE="https://ladish.org"
+SRC_URI="https://dl.ladish.org/ladish/${MY_P}.tar.bz2"
+S="${WORKDIR}/${MY_P}"
+
+# common/klist.h is linked list code borrowed from linux kernel, and thus is GPL2 only
+# otherwise it is GPL-2+ code
+LICENSE="GPL-2 GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+
+# media-sound/a2jmidid[dbus] is optional but recommended runtime-only dependency
+IUSE="debug +lash +a2jmidid"
+
+BDEPEND="
+	virtual/pkgconfig
+"
+RDEPEND="
+	dev-libs/expat
+	media-sound/jack2[dbus]
```

Comment:
Is that a fact? It is somewhat negating virtual/jack below. If it requires a dbus-enabled jack implementation, ruling out media-sound/jack-audio-connection-kit, would it work with `media-video/pipewire[dbus,jack-sdk]` alternatively?